consensus/state_processing/src/consensus_context.rs

```diff
@@ -153,16 +158,29 @@ impl<E: EthSpec> ConsensusContext<E> {
         state: &BeaconState<E>,
         attestation: &Attestation<E>,
     ) -> Result<&IndexedAttestation<E>, BlockOperationError<AttestationInvalid>> {
-        let key = (
-            attestation.data.clone(),
-            attestation.aggregation_bits.clone(),
-        );
+        let aggregation_bits = match attestation {
+            Attestation::Base(attn) => {
```

Comment:
hmmm not sure I understand the change in logic here?

Reply:
the `indexed_attestations` hashmap was originally defined as a 

```
HashMap<
        (
            AttestationData,
            BitList<E::MaxValidatorsPerCommittee>,
        ),
        IndexedAttestation<E>,
>
```

I updated the BitList portion to `BitList<E::MaxValidatorsPerCommitteePerSlot>`  so thats its compatible with the longer `aggregation_bits` electra variant

this bit of logic here extends the length of `aggregation_bits` for the base variant so that its compatible with the longer bitlist that the hashmap now expects.